health/statuscheck/model/status/status.pb.go

```diff
@@ -1,26 +1,26 @@
-// Code generated by protoc-gen-gogo.
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
This should have been generated by gogo. How did you execute the code generation?